src/blender_mcp/server.py

```
@@ -515,6 +587,218 @@ def get_hyper3d_status(ctx: Context) -> str:
+        logger.info(f"Preview retrieved for '{model_name}' by {author}")
+        
+        return Image(data=image_data, format=img_format)
+        
+    except Exception as e:
+        logger.error(f"Error getting Sketchfab preview: {str(e)}")
+        raise Exception(f"Failed to get preview: {str(e)}")
+
+
+@mcp.tool()
+def download_sketchfab_model(
+    ctx: Context,
```

Ruff (0.14.10)

Unused function argument: `ctx`

(ARG001)